cmake/linux/package_linux.sh.in

```diff
@@ -40,8 +39,15 @@ function skipped {
 	echo -e "   ${PLAIN}[${YELLOW}skipped${PLAIN}] ${1}"
 }
 
+# Run a command silently, but print output if it fails
+function try_quiet {
+    output="$("$@" 2>&1)" && return
+    printf '%s\n' "$output"
```

Comment:
Dumb question: What's the difference to

```suggestion
    echo "$output"
```

?

Reply:
That was just out of habit if `output="-e"` but that can never happen here. +1 `echo` for readability.

Reply:
Ah interesting!